### PR TITLE
Schedule export AD renewal letters

### PR DESCRIPTION
**Pull request by cintamani**
From: https://eaflood.atlassian.net/browse/RUBY-581

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -37,6 +37,7 @@ EXPIRE_REGISTRATION_EXEMPTION_RUN_TIME="00:05"
 # Exporter config
 EXPORT_SERVICE_BATCH_SIZE=10
 EXPORT_SERVICE_EPR_EXPORT_TIME="1:05"
+EXPORT_SERVICE_AD_RENEWAL_LETTERS_TIME="2:05"
```
Comment (cintamani) [inline .env.example]:
```suggestion
EXPORT_SERVICE_AD_RENEWAL_LETTERS_TIME="1:05"
```

File: .travis.yml [hunk 2]
```diff
@@ -21,6 +21,7 @@ env:
     - EXPORT_SERVICE_BULK_EXPORT_FREQUENCY="friday"
     - EXPORT_SERVICE_BULK_EXPORT_TIME="16:05"
     - EXPORT_SERVICE_BOXI_EXPORT_TIME="03:05"
+    - EXPORT_SERVICE_AD_RENEWAL_LETTERS_TIME="2:05"
```
Comment (cintamani) [inline .travis.yml]:
```suggestion
    - EXPORT_SERVICE_AD_RENEWAL_LETTERS_TIME="1:05"
```

File: config/schedule.rb [hunk 3]
```diff
@@ -50,6 +50,13 @@
   rake "reports:export:boxi"
 end
 
+# This is the daily AD renewal letters export service.
+# Will run once a day in the early morning hours and generate a zip file containing
+# data required for boxi.
+every :day, at: (ENV["EXPORT_SERVICE_AD_RENEWAL_LETTERS_TIME"] || "02:05"), roles: [:db] do
```
Comment (cintamani) [inline config/schedule.rb]:
```suggestion
every :day, at: (ENV["EXPORT_SERVICE_AD_RENEWAL_LETTERS_TIME"] || "01:05"), roles: [:db] do
```

File: .env.example [hunk 4]
```diff
@@ -37,6 +37,7 @@ EXPIRE_REGISTRATION_EXEMPTION_RUN_TIME="00:05"
 # Exporter config
 EXPORT_SERVICE_BATCH_SIZE=10
 EXPORT_SERVICE_EPR_EXPORT_TIME="1:05"
+EXPORT_SERVICE_AD_RENEWAL_LETTERS_TIME="1:05"
```
Comment (irisfaraway) [inline .env.example:40]: Is it OK to have this scheduled at the same time as the EPR export? Feels like they shouldn't be clashing.

File: config/schedule.rb [hunk 5]
```diff
@@ -50,6 +50,13 @@
   rake "reports:export:boxi"
 end
 
+# This is the daily AD renewal letters export service.
+# Will run once a day in the early morning hours and generate a zip file containing
+# data required for boxi.
```
Comment (irisfaraway) [inline config/schedule.rb]:
```suggestion
# all AD renewal letters expiring in X days.
```

File: spec/schedule_spec.rb [hunk 6]
```diff
@@ -63,6 +63,13 @@
     expect(job_details[:every][1][:at]).to eq(ENV["EXPIRE_REGISTRATION_EXEMPTION_RUN_TIME"])
   end
 
+  it "takes the expire ad renewal letters execution time from the appropriate ENV variable" do
```
Comment (irisfaraway) [inline spec/schedule_spec.rb]:
```suggestion
  it "takes the export AD renewal letters execution time from the appropriate ENV variable" do
```

File: config/schedule.rb (hunk 5 above)
Comment (irisfaraway) [reply config/schedule.rb]: The zip file bit may be inaccurate as well – not sure

**Review by irisfaraway — CHANGES_REQUESTED** [review]
A few bits, but nothing major 👍

File: config/schedule.rb (hunk 5 above)
Comment (cintamani) [reply config/schedule.rb]: Copy paste fail! D:

File: .env.example (hunk 4 above)
Comment (cintamani) [reply .env.example:40]: The EPR exports in prod actually run at 22.05 :) We should fix these values to reflect the production once. For another PR :)

File: config/schedule.rb [hunk 7]
```diff
@@ -50,6 +50,13 @@
   rake "reports:export:boxi"
 end
 
+# This is the daily AD renewal letters export service.
+# Will run once a day in the early morning hours and generate a zip file containing
```
Comment (cintamani) [inline config/schedule.rb]:
```suggestion
# Will run once a day in the early morning hours and generate a PDF file containing
```

**Review by irisfaraway — APPROVED** [review]

**Pull request merged**